### PR TITLE
Beta test fix

### PR DESCRIPTION
**Pull request by gull192**
fixed: 
 - keyboard options 
 - save user data after change
 - save user info after auth flow

### CODE REVIEW

File: composeApp/src/commonMain/kotlin/band/effective/office/elevator/data/repository/ProfileRepositoryImpl.kt
Comment (UserNameMax) [inline composeApp/src/commonMain/kotlin/band/effective/office/elevator/data/repository/ProfileRepositoryImpl.kt:1]: We have opportunity update employee email?

File: composeApp/src/commonMain/kotlin/band/effective/office/elevator/domain/models/User.kt
Comment (UserNameMax) [inline composeApp/src/commonMain/kotlin/band/effective/office/elevator/domain/models/User.kt:1]: if user not employee, how he log in and he can log in?

File: composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/authorization_profile/AuthorizationProfileScreen.kt [hunk 1]
```diff
@@ -95,6 +99,9 @@ fun AuthorizationProfileComponent(
     val borderColor2 = remember { mutableStateOf(textGrayColor) }
     val leadingColor2 = remember { mutableStateOf(textGrayColor) }
 
+    var personName by remember { mutableStateOf(state.name) }
```
Comment (UserNameMax) [inline composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/authorization_profile/AuthorizationProfileScreen.kt:102]: If we change name in state we can collect it change.  Why we need remember?

File: composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/store/AuthorizationStoreImpl.kt [hunk 2]
```diff
@@ -41,18 +41,16 @@ class AuthorizationStoreFactory(
...
-                    }
-                    is AuthorizationStore.Intent.ChangeEmail -> dispatch(Msg.ChangeEmail(intent.email))
-                    is AuthorizationStore.Intent.ChangeName -> dispatch(Msg.ChangeName(intent.name))
-                    is AuthorizationStore.Intent.ChangePhoneNumber -> dispatch(Msg.ChangePhoneNumber(intent.phoneNumber))
-                    is AuthorizationStore.Intent.ChangePost -> dispatch(Msg.ChangePost(intent.post))
-                    is AuthorizationStore.Intent.ChangeTelegram -> dispatch(Msg.ChangeTelegram(intent.telegram))
-                    is AuthorizationStore.Intent.UpdateUserInfo -> dispatch(Msg.ChangeUser(intent.user))
+            when (intent) {
+                AuthorizationStore.Intent.ContinueButtonClicked -> {
+                    TODO()
```
Comment (UserNameMax) [inline composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/store/AuthorizationStoreImpl.kt:46]: Remove todo or add description

**Review by UserNameMax — APPROVED** [review]

File: composeApp/src/commonMain/kotlin/band/effective/office/elevator/data/repository/ProfileRepositoryImpl.kt
Comment (gull192) [reply composeApp/src/commonMain/kotlin/band/effective/office/elevator/data/repository/ProfileRepositoryImpl.kt:1]: Technically, no, the backend is waiting for it in response

File: composeApp/src/commonMain/kotlin/band/effective/office/elevator/domain/models/User.kt
Comment (gull192) [reply composeApp/src/commonMain/kotlin/band/effective/office/elevator/domain/models/User.kt:1]: no way

File: composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/authorization_profile/AuthorizationProfileScreen.kt (hunk 1 above)
Comment (gull192) [reply composeApp/src/commonMain/kotlin/band/effective/office/elevator/ui/authorization/authorization_profile/AuthorizationProfileScreen.kt:102]: fix due to a problem from here: fix due to a problem from [here](https://medium.com/androiddevelopers/effective-state-management-for-textfield-in-compose-d6e5b070fbe5)

**Pull request merged**